src/common/Throttle.cc

```diff
@@ -760,14 +814,13 @@ void TokenBucketThrottle::add_tokens() {
 }
 
 void TokenBucketThrottle::schedule_timer() {
-  add_tokens();
-
   m_token_ctx = new FunctionContext(
       [this](int r) {
         schedule_timer();
       });
+  m_timer->add_event_after(m_schedule_tick / 1000.0, m_token_ctx);
 
-  m_timer->add_event_after(1, m_token_ctx);
+  add_tokens();
```

Comment:
why move add_tokens() after add_event_after()?

Reply:
`add_tokens()` will take time. So, for good accuracy, we should `add_event` first.